platform/debian/koreader.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
As an aside, see https://github.com/koalaman/shellcheck/issues/880#issuecomment-298032886

But since KOReader isn't supposed to be run as root that shouldn't be a concern here.

I figure you might also want to change this one:

https://github.com/koreader/koreader/blob/0a2a1fffe8305555ac431c1926c16c1a0a15af3d/kodev#L1

Reply:
Thanks for the information. I've also changed kodev.

Reply:
Thanks!